tests/test_pp_adj_prop.py

```diff
@@ -0,0 +1,452 @@
+import pytest
+import numpy as np
+import scipy as sp
+import pandas as pd
+from scipy import sparse
+from .test_method_score_cell_main import load_toy_data
+from anndata import AnnData
+import scdrs
+
+
+def load_toy_data_adj_prop():
+    """
+    Toy data for checking `adj_prop`
```

Comment:
1. Could add a description of the toy data set. Is the following correct?

> Simulate a data set with 2 cell types:
> ct1 (n=100): genes 1-50 are i.i.d. normal with mean 1, variance 4; genes 51-100 are i.i.d. normal with mean 0, variance 1
> ct2 (n=900): genes 1-50 are i.i.d. normal with mean 0, variance 1; genes 51-100 are i.i.d. normal with mean 1, variance 4
> Also a balanced data set with all ct1 cells (n = 100) and the first 100 ct2 cells (n = 100)

2. Is the following simulated data set more helpful? simulate two data sets  (a) the original data set with 100 ct1 cells and 200 ct2 cells (b) the repeated data set with ct1 cells each replicated XX times (and associated with 1/XX weights), with ct2 cells each replicated YY times (and associated with 1/YY weights). And then test the consistency of each function

Reply:
1. Make sense; will update. rand is Uniform[0,1] instead of normal
2. The weights are automatically inferred so I don't think we want to simulate that. Identically replicating cells does seem to be a better idea for a more controlled test. I will add it to the wish list and maybe update it later.